llvm/utils/TableGen/Common/CodeGenSchedule.cpp

```diff
@@ -979,8 +979,9 @@ unsigned CodeGenSchedModels::addSchedClass(const Record *ItinClassDef,
     return SC.isKeyEqual(ItinClassDef, OperWrites, OperReads);
   };
 
-  auto I = find_if(make_range(schedClassBegin(), schedClassEnd()), IsKeyEqual);
```

Comment:
I considered using schedClases() here but I would still need schedClassEnd() on the next line. The rest of this function already uses the SchedClasses member directly so I decided to just skip the accessor methods.